app/webhook.go

```diff
@@ -323,13 +323,12 @@ func (a *App) CreateIncomingWebhookForChannel(creatorId string, channel *model.C
 		return nil, model.NewAppError("CreateIncomingWebhookForChannel", "api.incoming_webhook.invalid_username.app_error", nil, "", http.StatusBadRequest)
 	}
 
-	if result := <-a.Srv.Store.Webhook().SaveIncoming(hook); result.Err != nil {
```

Comment:
Prabably here makes sense to return just `return a.Srv.Store.Webhook().SaveIncoming(hook)`

Reply:
Can't believe I didn't notice that. Thanks.

Reply:
No problem :)